inst/@sym/size.m

```diff
@@ -65,7 +65,7 @@
 
   % Note: symbolic sized matrices should return double, not sym/string.
 
-  n = x.size;
+  n = x.symsize;
```

Comment:
This was required. Because if I used `size` as property in the `sym` class then too (weirdly) octave called the `size` function and did not use the property..